### PR TITLE
[GRAFANA] Introduced GF_SECURITY_ADMIN_PASSWORD__FILE

### PR DESCRIPTION
**Pull request by etiennejournet**
It would be good if we could use "GF_SECURITY_ADMIN_PASSWORD__FILE" to set password using files (useful when we use vault-agent-injector)

### CODE REVIEW

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 4.2.1
+version: 4.3.1
```
Comment (zanhsieh) [inline stable/grafana/Chart.yaml]: This needs to bump again.

Comment (etiennejournet) [reply stable/grafana/Chart.yaml]: Do you want a 4.4.0 ?

**Comment by zanhsieh** [comment]
@etiennejournet 
Could you bump the chart version again? This chart version advances fast.

**Comment by etiennejournet** [comment]
> @etiennejournet
> Could you bump the chart version again? This chart version advances fast.

As of today the chart version is 4.3.2. I bumped it to 4.3.3

**Comment by zanhsieh** [comment]
@etiennejournet 
Could you bump again? Sorry for advancing too fast.

**Comment by etiennejournet** [comment]
dont merge i want to add something

**Comment by etiennejournet** [comment]
ready to merge

**Comment by zanhsieh** [comment]
Humm.. Mind you fix the conflicting file listed below?

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**